src/components/content/injection.tsx

```diff
@@ -27,10 +27,11 @@ export function Injection({ href }: InjectionProps) {
         else return res.text()
       })
       .then((data) => {
-        if (data.contentType && data.data) {
-          setValue(data.data.value)
-          if (data.data.license) {
-            setLicense(data.data.license)
+        const returnData = data as EntityProps
```

Comment:
Like last time: you can move that to the argument itself: `then((data: EntityProps)`